test/parallel/test-internal-module-require.js

```diff
@@ -87,6 +87,7 @@ if (process.argv[2] === 'child') {
         });
       } else {
         require(id);
+        require(`node:${id}`);
```

Comment:
```suggestion
        require(`node:${id}`);
```

Wouldn't this be enough?

Reply:
no, because then it's not validating that you can require it *without* the protocol.